invenio.cfg

```diff
@@ -162,6 +169,12 @@ ZENODO_LEGACY_SEARCH_MAP = {
     "title": "metadata.title",
     "type": "metadata.resource_type.props.type",
     "version": "versions.index",
+    "part_of.title": "custom_fields.imprint\:imprint.title",
```

Comment:
question: why `part_of`?

Reply:
That's how this field was implemented in zenodo.

See the [search guide](https://help.zenodo.org/guides/search/) or the [jsonschema](https://github.com/zenodo/zenodo/blob/d910dbc0c29f9829ca24a2239f4f01cf295c9687/zenodo/modules/records/jsonschemas/records/record-v1.0.0.json#L848) .

Even though in the UI all these fields (`imprint` and `part_of`) are together, in the backend they were split back in Zenodo. After discussing IRL, we decided to merge them under a single custom field `imprint` and guarantee retro-compatibility for the search.